### ISSUE TITLE
Make a STABLE release (the current version is from 9/8-2017, almost a decade old)

### ISSUE THREAD

**Issue by tveyben** [greenshot/greenshot#562]
Greenshot is available for installation via the package manager [Chocolatey!](https://community.chocolatey.org/packages/greenshot)
Only the STABLE release is available, thus please make a new and recent "STABLE" version of Greenshot so the latest and greatest features can be installed and hence used without having to do a manual install.

Why?
- I use choco for as much PC setup as possible
- I use keyboard shortcuts as much as possible
- I use Greenshot a lot for adding documentation to Jira tickets etc.

One of the great features that's only available in the UNSTABLE branch is "FEATURE-1125: Add shortcuts (0-9, +/-) for foreground color, background color, line thickness, bold and shadow (#338)"

The STABLE version (1.2.10.6) is from 9 August 2017, soon a decade old.

So for now I manually have to update all my PC's to this version, but I would prefer to be able to handle this via Choco.

* One option is to get a "UNSTABLE" package added to the Choco repo
  * I have sent an e-mail to the choco maintainer suggesting this
* Another option is to make a new STABLE release
  * Hence this ticket :-)

:-) TvE

**Comment by tobias9931** [comment]
I would also be very happy about a new stable version.
The last stable is from 2017.
For Windows 11.
With a fix for #546

**Comment by michalpl7** [comment]
Would be great if You finally release stable update, I asked some time about more arrow styles - https://github.com/greenshot/greenshot/issues/442 - Thnx.

**Comment by AgentSam** [comment]
Yes, please. Pretty please.

**Comment by drauch** [comment]
Would also gladly install the newest stable version :-)

**Comment by c0nsaw** [comment]
Please do, few years ago our company removed the old stable version from our approved software allowed due to it failing corporate security scans, we are forced to use snagit now, which is awful. 

An updated, secure, open source version would be allowed again

**Comment by tveyben** [comment]
Ping @larsvegas0815

**Comment by FF-Brown** [comment]
idk about the rest of you but if I were an admin of an open source software, I bet donations would be a good way to get me to put out a new release 😉

**Comment by tobias9931** [comment]
My company would make a donation to @larsvegas0815 
We have problems installing the latest releases under Windows 11.
Everything works with Windows 10

**Comment by alex-jensen-wb** [comment]
> My company would make a donation to [@larsvegas0815](https://github.com/larsvegas0815) [@Lakritzator](https://github.com/Lakritzator) We have problems installing the latest releases under Windows 11. Everything works with Windows 10

https://getgreenshot.org/support/

**Comment by tobias9931** [comment]
> > My company would make a donation to [@larsvegas0815](https://github.com/larsvegas0815) [@Lakritzator](https://github.com/Lakritzator) We have problems installing the latest releases under Windows 11. Everything works with Windows 10
> 
> https://getgreenshot.org/support/

currently looks like a dead letterbox to me

**Comment by JamesPratt-LWL** [comment]
You can always download the source and compile it yourself in Visual Studio? I just did and my version is 1.3.280-UNSTABLE and it works fine on Windows 11 for me? I do not see any reference to it in HKLM either? Am I missing something?

**Comment by luc0x61** [comment]
> You can always download the source and compile it yourself in Visual Studio? I just did and my version is 1.3.280-UNSTABLE and it works fine on Windows 11 for me? I do not see any reference to it in HKLM either? Am I missing something?

1. Assume I'm not a developer, or
2. Assume I'm not a VS developer, or
3. Assume I'm not allowed installing/using VS at my place, or
4. Assume I prefer using applications compiled/tested/signed by someone more competent than me, or
5. Assume there could be any other reason on why it's been **already** packaged as an installer, ages ago

Then an updated package installer could be welcomed

**Comment by Neustradamus** [comment]
I have informed a moment ago @jklingen about version numbers.

1.1/1.3/1.5/1.7/1.9 branches are for development (unstable), and 1.0/1.2/1.4/1.6/1.8 branches are stable.

Normally, when it will be stable, a 1.4 version will be here.

**Comment by jklingen** [comment]
@Neustradamus yes, and [I answered](https://github.com/greenshot/greenshot/issues/579#issuecomment-2909363365)

> I think just incrementing "normally" is fine for most users and switching the versioning scheme now could be quite confusing for many folks.

**Greenshot 1.3.190 is the first stable release for Greenshot 1.3**. (in the meantime, .192 has been released for a bugfix). You'll get that when you download from our [website](https://getgreenshot.org/downloads/), and it is [marked in Github as the latest release](https://github.com/greenshot/greenshot/releases).


> Normally, when it will be stable, a 1.4 version will be here.

There are different versioning schemes, we're obviously not using the one you prefer.
